src/components/facets.tsx

```diff
@@ -81,96 +81,135 @@ export const stringify = (
   return qsStringify({ ...rest, [queryStringKey]: facetString });
 };
 
-type Props = {
+type FacetProps = {
   /**
    * The facet data to be displayed
    */
-  data: Facet[];
+  data: FacetObject;
   /**
-   * Extra components to be added in the "action" area, map of <facet name, component>
+   * Extra components to be added in the "action" area
    */
-  extraActionsFor?: Map<string, ReactNode>;
+  extraActions?: ReactNode;
   /**
    * Key with which to add the facets in the querystring (defaults to "facets")
    */
   queryStringKey?: string;
 };
 
-const Facets: FC<Props & HTMLAttributes<HTMLDivElement>> = ({
+export const Facet: FC<FacetProps & HTMLAttributes<HTMLDivElement>> = ({
   data,
-  extraActionsFor,
+  extraActions,
   queryStringKey = 'facets',
-  className,
   ...props
 }) => {
   const location = useLocation();
+  const search = parse(location.search, queryStringKey);
 
-  if (!(data && data.length)) {
+  if (!data.values?.length) {
     return null;
   }
+  return (
+    <div {...props}>
```

Comment:
What extra props are expected here? Is it just `className`?

Reply:
That could be anything really, including `className`
It's just to avoid wrapping this component in an unnecessary div if we ever need to add an `aria-` or `data-` attribute or anything else really.
We already started having this pattern accross new and modified Franklin components